slices/procps.yaml

```diff
@@ -0,0 +1,30 @@
+package: procps
+
+slices:
+  bins:
+    essential:
+      - init-system-helpers_bins
+      - libc6_libs
+      - libncursesw6_libs
+      - libproc2-0_libs
+      - libsystemd0_libs
+      - libtinfo6_libs
+    contents:
+      /usr/bin/free:
+      /usr/bin/kill:
+      /usr/bin/pgrep:
+      /usr/bin/pidwait:
+      /usr/bin/pkill: {symlink: pgrep}
+      /usr/bin/pmap:
+      /usr/bin/ps:
+      /usr/bin/pwdx:
+      /usr/bin/skill:
+      /usr/bin/slabtop:
+      /usr/bin/snice: {symlink: skill}
+      /usr/bin/tload:
+      /usr/bin/top:
+      /usr/bin/uptime:
+      /usr/bin/vmstat:
+      /usr/bin/w:
+      /usr/bin/watch:
+      /usr/sbin/sysctl:
```

Comment:
etc files and `/usr/lib/sysctl.d/99-protect-links.conf` are missing. better put them in a config slice.